docs/examples/README.md

```diff
@@ -0,0 +1,60 @@
+# Get started with examples
+
+Learn how to install Playwright, set up your dev environment to author Playwright scripts, and example recipes to bootstrap your scripts.
+
+## Installing Playwright
+
+Playwright is a Node.js library and can be acquired through the npm registry. Use npm or yarn to install Playwright in your Node.js project.
+
+```
+npm i playwright
+```
+
+Once installed, you can `require` Playwright in your Node.js scripts. 
+
+```js
+const playwright = require('playwright');
```

Comment:
We recommend destructuring everywhere: `const { chromium } = require('playwright');`.